rootfs/conf/kops/helmfile.yaml

```diff
@@ -22,9 +22,9 @@ repositories:
 - name: "stable"
   url: "https://kubernetes-charts.storage.googleapis.com"
 
-# Cloud Posse incubator repo of helm charts
```

Comment:
Why are you deleting the `cloudposse-incubator` repo?

Reply:
ups